keras_contrib/applications/densenet.py

```diff
@@ -132,13 +132,13 @@ def DenseNet(depth=40, nb_dense_block=3, growth_rate=12, nb_filter=16, nb_layers
                 (bottleneck is False) and (reduction == 0.0) and (dropout_rate == 0.0) and (weight_decay == 1E-4):
             # Default parameters match. Weights for this model exist:
 
-            if K.image_dim_ordering() == 'th':
+            if K.image_data_format() == 'channels_first':
                 if include_top:
-                    weights_path = get_file('densenet_40_12_th_dim_ordering_th_kernels.h5',
+                    weights_path = get_file('densenet_40_12_th_data_format_th_kernels.h5',
```

Comment:
No I did not upload any new file to a new path. The path remains the same, just the name of the weight file changes when it is downloaded onto a local machine.